### DISCUSSION TITLE
failed  to install in anaconda

### DISCUSSION THREAD

**Discussion by Vampire-2020** [danielhrisca/asammdf#612] [Ideas]
Hi
Thanks for your work.
I downloaded your code and tried to install in anaconda but failed.
it said that 'Type Error:stat:path should be string,bytes, os.PathLike or integer not NonType.'
What should I do to deal with this?
Hope for your help,
Thanks a lot.

**Comment by JulienGrv** [comment]
Hi,

Please show us your code
